File: src/scripts/scrape_hotels.py
import scrapling
import asyncio

async def scrape_hotels():
    response = await scrapling.run({
        'url': 'https://www.google.com/maps/search/oteller+Konya',
        'method': 'get',
        'options': {
            'maxResults': 10,
        }
    })

    print('Top 10 hotels in Konya:', response)

if __name__ == "__main__":
    asyncio.run(scrape_hotels())

File: src/scripts/scrape-hotels.ts
import { scrapling } from 'scrapling';

async function scrapeHotels() {
  const results = await scrapling.run({
    url: 'https://www.google.com/maps/search/oteller+Konya',
    method: 'get',
    options: {
      maxResults: 10,
    }
  });

  console.log('Top 10 hotels in Konya:', results);
}

scrapeHotels().catch(e => {
  console.error(e);
  process.exit(1);
});